Digipost.Signature.Api.Client.Direct.Tests/Smoke/DirectClientSmokeTests.cs

```diff
@@ -16,12 +14,76 @@ public class DirectClientSmokeTests : SmokeTests
     {
         private static DirectClient _directClient;
 
-        private static StatusReference _statusReference;
-        private static ConfirmationReference _confirmationReference;
-        private static XadesReference _xadesReference;
-        private static PadesReference _padesReference;
+        private static TestHelper _t;
 
-        protected static DirectClient GetDirectClient()
+        [ClassInitialize]
+        public static void Setup(TestContext context)
+        {
+            _t = new TestHelper(GetDirectClient());
+        }
+
+        [TestMethod]
+        public void Can_create_job_with_one_signer()
```

Comment:
This fluid test-pattern is readable and efficient! This should be the way to do all smoke tests 👍